ANMN/LTSP/TSaggregator/TSagg_metadata.json

```diff
@@ -46,6 +46,41 @@
             "standard_name": "depth status_flag", 
             "_FillValue": 99
             },
+        "PRES": {
+            "ancillary_variables": "PRES_quality_control",
+            "long_name": "sea_water_pressure_due_to_sea_water",
+            "standard_name": "sea_water_pressure_due_to_sea_water",
+            "units": "dbar",
+            "valid_max": 12000.0,
+            "valid_min": -15.0,
+            "_FillValue": 999999.0 
+            },
+        "PRES_quality_control": {
+            "flag_meanings": "No_QC_performed Good_data Probably_good_data Bad_data_that_are_potentially_correctable Bad_data Value_changed Not_used Not_used Not_used Missing_value",
+            "flag_values": "0, 1, 2, 3, 4, 5, 6, 7, 8, 9",
+            "long_name": "quality flag for sea_water_pressure_due_to_sea_water",
+            "quality_control_conventions": "IMOS standard flags",
+            "standard_name": "depth status_flag", 
+            "_FillValue": 99
+            },
+        "PRES_REL": {
+            "ancillary_variables": "PRES_REL_quality_control",
+            "applied_offset": -10.135297, 
```

Comment:
I've checked few files and the value is the same but it is a good point. I will check it in deep.

Reply:
We have a problem here.
There are 4104 files in AODN THREDDS server with `PRES_REL` in the variable list. 3523 have `applied_offset` attribute and there are, as guessed, non identical:
count    3523.000000
mean      -10.247914
std         2.130044
min       -94.903198
25%       -10.135297
50%       -10.135297
75%       -10.135297
max         3.777505

One solution could be to add a global attribute `PRES_REL_applied_offset` with a list of the values from each instrument

Reply:
The vertical bar is the median.

![image](https://user-images.githubusercontent.com/2551823/62339577-d0034680-b51f-11e9-9c9d-1c9ffd8e242f.png)

Reply:
How many files are there where the offset is *not* exactly -10.135297, and how old are those files (in terms of date_created) ?
If it's a significant fraction, maybe we need to make an exception and do a quick adjustment during aggregation

Reply:
75% of all the values are -10.135297. Below the freq table. The rest of the values have only one occurrence.

-10.135297    2638
-10.132500     772
-10.132500      32
-10.135297      25

Regarding the publication date of the files, most of the variability of the `applied_offset` values came from 2014 and 2017, some from 2016. I think we cannot generalise.

![image](https://user-images.githubusercontent.com/2551823/62432735-a98a1900-b774-11e9-8efc-0f7cb3f62817.png)



Here is a sample of the `applied_offset` attribute for `PRES_REL` with not-present (filled with `FILLVALUE`) and present values of the offset

```
float PRES_REL(OBSERVATION=1521894);
  :_FillValue = 999999.0f; // float
  :ancillary_variables = "PRES_REL_quality_control";
  :long_name = "sea_water_pressure_due_to_sea_water";
  :standard_name = "sea_water_pressure_due_to_sea_water";
  :units = "dbar";
  :valid_max = 12000.0; // double
  :valid_min = -15.0; // double
  :applied_offset = 999999.0, 999999.0, 999999.0, 999999.0, -10.135296821594238; // double
```

Reply:
changed to `applied_offset_by_instrument` and `NaN`s instead of `FILLVALUE`

```
float PRES_REL(OBSERVATION=1521894);
  :_FillValue = 999999.0f; // float
  :ancillary_variables = "PRES_REL_quality_control";
  :long_name = "sea_water_pressure_due_to_sea_water";
  :standard_name = "sea_water_pressure_due_to_sea_water";
  :units = "dbar";
  :valid_max = 12000.0; // double
  :valid_min = -15.0; // double
  :applied_offset_by_instrument = NaN, NaN, NaN, NaN, -10.135296821594238; // double
```